BunqSdk.Tests/Model/Generated/Endpoint/PaymentTest.cs

```diff
@@ -15,19 +15,19 @@ public class PaymentTest : BunqSdkTestBase
         /// <summary>
         /// Config values.
         /// </summary>
-        private const string PAYMENT_AMOUNT_EUR = "0.01";
-        private const string PAYMENT_CURRENCY = "EUR";
-        private const string PAYMENT_DESCRIPTION = "C# test Payment";
+        private const string Amount = "0.01";
+        private const string Currency = "EUR";
+        private const string Description = "C# test Payment";
```

Comment:
Please rename this back to `PaymentDescription`. This is our standard on how to name constants. There should be a `group` identifier and in this case it is `Payment`.